src/commands/push-to-app-collection.yaml

```diff
@@ -56,7 +56,16 @@ steps:
   - run: |
         cd .app-collection && git commit -m "$(cat ../.git_commit_message)"
   - run: |
-        cd .app-collection && git push || (echo "Error pushing app to the collection. See known errors in https://github.com/giantswarm/architect-orb/blob/master/README.md#push-to-app-collection" && exit 1)
+        cd .app-collection && \
+        declare -i attempt=0 && \
+        while ! git push; do
```

Comment:
I think when you push and fail and push again it will continue to fail. We need to pull and push and when it fails try again to pull and push. Further I found `Error pushing app to the collection.` good to see so let's maybe keep that bit.

Reply:
Ah, OK, I'll work that in. Thanks.

Reply:
Hey @xh3b4sd, should we keep the link to README and the explanation about re-trying manually?

Reply:
PTAL